test/common.bash

```diff
@@ -155,15 +155,6 @@ env_require CH_BUILDER
 if [[ $CH_BUILDER == ch-grow ]]; then
     env_require CH_GROW_STORAGE
 fi
-# Set path to the right Charliecloud. This uses a symlink in this directory
-# called "bin" which points to the corresponding bin directory, either simply
-# up and over (source code) or set during "make install".
-#
-# Note that sudo resets $PATH, so if you want to run any Charliecloud stuff
-# under sudo, you must use an absolute path.
-ch_bin="$(cd "$(dirname "${BASH_SOURCE[0]}")/bin" && pwd)"
-ch_bin="$(readlink -f "${ch_bin}")"
```

Comment:
OK, so this `readlink` was already in there. I wonder why ....

Reply:
I believe the readlink _here_ was to make sure that the test/bin symlink was valid.